### PR TITLE
[FEATURE] Update Polling Url by Latest updatedAt

### PR DESCRIPTION
**Pull request by hserang**
## Purpose
Find the latest updatedAt in polling response to update polling url

## Status
Ready, but may need to investigate gatewayd's handling of timezones when returning createdAt and updatedAt

### CODE REVIEW

File: app/scripts/modules/payments/collections/payments.js [hunk 1]
```diff
@@ -104,22 +104,41 @@ var Payments = Backbone.Collection.extend({
...
   },
 
-  getNewTransactionsUrl: function(updatedAt) {
-    var timeStamp = encodeURIComponent(moment(updatedAt).format('YYYY-MM-DD HH:mm:ss.ms'));
+  setLatestPayment: function(payments) {
+    var newestUpdatedPayment;
+
+    // todo: can't rely on response sorting at the moment
+    // look into timestamp issues later
+    if (payments.length) {
```
Comment (AiNoKame) [inline app/scripts/modules/payments/collections/payments.js]: This is short circuited on line 152, in the success callback of the fetch. Do you still need this check?

File: app/scripts/modules/payments/collections/payments.js [hunk 2]
```diff
@@ -104,22 +104,41 @@ var Payments = Backbone.Collection.extend({
...
+      });
+
+      this.latestPayment = newestUpdatedPayment.updatedAt;
+    } else {
+      this.latestPayment = this.at(0).get('updatedAt');
+    }
+  },
+
+  getNewTransactionsUrl: function() {
+    var timeStamp = encodeURIComponent(moment(this.latestPayment).format('YYYY-MM-DD HH:mm:ss.ms'));
```
Comment (AiNoKame) [inline app/scripts/modules/payments/collections/payments.js]: Is this.latestPayment defined somewhere? The getNewTransactionsUrl method uses this.latestPayment but is called before fetching the new transactions (where this.latestPayment is originally set in its success callback). Prior to this, it'll be undefined, which is fed into moment. Maybe call setLatestPayment in the fetchRippleTransactions method?

File: app/scripts/modules/payments/collections/payments.js (hunk 1 above)
Comment (hserang) [inline app/scripts/modules/payments/collections/payments.js]: This check is required - it also allows us to return the latest updatedAt stamp from the collection in the case that there is no params passed in.

**Comment by AiNoKame** [comment]
LGTM!

**Pull request merged**